### ISSUE TITLE
Differing jurisdiction in event participant is disregarded

### ISSUE THREAD

**Issue by kwa20** [hzi-braunschweig/SORMAS-Project#7169]
### Bug Description

If the jurisdiction of the event participant differs from the event jurisdiction, users in the event jurisdiction can still fully edit the event participant outside of their jurisdiction.
### Steps to Reproduce

1. Create an event in region1 and district1 with e.g. a national user
2. Add an event participant and set their jurisdiction to region2 and district2
3. Log in as a supervisor or officer in jurisdiction region1/district1
4. Open the event participant tab and check the particpant

### Expected Behavior

The supervisor/officer has full access to the event but read only access to the event participant.

### Screenshots

TestAccess2 user jurisdiction differs from ep jurisdiction
<img width="552" alt="AccessEp" src="https://user-images.githubusercontent.com/70317594/140529611-03a74b39-f1b9-4d56-af67-71819c377a62.PNG">

### System Details

* Device:
* SORMAS version: 1.64.1
* Android version/Browser: chrome
* Server URL: test1401.sormas-oegd.de
* User Role: Surveillance/Contact Supervisor, Surveillace/Contact Officer

**Comment by MateStrysewske** [comment]
@kwa20 I don't think that's a bug but rather a conscious design decision because users who manage events need to be able to manage all its event participants. The additional event participant jurisdiction was added to make sure that users from a jurisdiction that differs from the event jurisdiction could still manage event participants that live/otherwise belong to their own jurisdiction. I believe this request should be discussed again.

**Comment by kwa20** [comment]
@MateStrysewske Fair enough, I also noticed that some servers require region and district to be compuslory in the event and other don't. I can't quite remember why that is anymore.

**Comment by bernardsilenou** [comment]
@MateStrysewske 
- I agree with @kwa20 specification and  think this is a bug, I can trace the initial specification issue and tag it here later. Here is the summary:
1. If the ep has no jurisdiction (NULL), yes users in the jurisdiction of the event has full access to the ep because they inherit the jurisdiction of the event.
2. if ep has a jurisdiction and  it is different from that of the event:
    2.1 Users in jurisdiction of the event has read access to the ep
    2.2  The reporting user (in jurisdiction of event  or (neither in jurisdiction of event nor ep))  has full access to the the ep  
   2.3  Users in jurisdiction of the ep has full access to the ep
    2.4 Users (not in jurisdiction of the event  or ep) and (are not reporting user)  has pseodonimized read access to the ep
2.5 Users not in jurisdiction of the event has read access to the event
2.5 No user has pseudo access to event
